google/cloud/testing_util/example_driver.cc

```diff
@@ -61,10 +61,14 @@ int Example::Run(int argc, char const* const argv[]) try {
 
   std::string const command_name = argv[1];
   auto command = commands_.find(command_name);
-  if (commands_.end() == command) {
+  if (commands_.end() == command && command_name != "--help") {
     PrintUsage(argv[0], "Unknown command: " + command_name);
     return 1;
   }
+  else if(commands_.end() == command && command_name == "--help") {
+    PrintUsage(argv[0], "");
```

Comment:
This is the first time we'll be passing an empty `msg` to `PrintUsage()`, which means it will output a blank line.  We should probably avoid that somehow.

Reply:
Done